File: Atcoder/PHP/190.ABC/B.php
<?php
fscanf(STDIN, "%d%d%d", $evis,$jap,$evija);

$evisjap = 0 ;
for($i=0;$i<$evis;$i++){
  fscanf(STDIN, "%d%d", $mori,$yama);
 if($mori<$jap && $yama > $evija) {
   $evisjap += 1;
  }
}

if($evisjap == 0){
 echo "No"; } else {
 echo "Yes";}

File: Atcoder/Ruby/190.ABC/B.rb
evisjap,evis,jap = gets.split(" ").map &:to_i

wakuwaku = 0
evisjap.times do
  kes,mori = gets.split.map(&:to_i)
  if(evis > kes && jap < mori)
    wakuwaku += 1
  end
end

if(wakuwaku > 0 )
  puts 'Yes'
else
  puts 'No'
end  
